lib/Conversion/TritonGPUToLLVM/TritonGPUToLLVMBase.h

```diff
@@ -1,6 +1,7 @@
 #ifndef TRITON_CONVERSION_TRITONGPU_TO_LLVM_BASE_H
```

Comment:
Do we really need this file?

Reply:
`ConvertTritonGPUOpToLLVMPattern` in this header file is needed by every conversions in every separated files.